src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -133,11 +137,20 @@ public void initForViewByQuestion(InstructorAttributes instructor,
         Map<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>> questionToResponseMap = bundle.getQuestionResponseMap();
         questionPanels = new ArrayList<InstructorFeedbackResultsQuestionTable>();
         
+        // if there is more than one question, we omit generation of responseRows,
+        // and load them by ajax question by question
+        boolean isLoadingStructureOnly = questionToResponseMap.size() > 1;
+                                        
         for (Map.Entry<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>> entry : questionToResponseMap.entrySet()) {
             FeedbackQuestionAttributes question = entry.getKey();
             List<FeedbackResponseAttributes> responses = entry.getValue();
             
-            questionPanels.add(buildQuestionTableAndResponseRows(question, responses, ""));
+            InstructorFeedbackResultsQuestionTable questionPanel = buildQuestionTableAndResponseRows(question, responses, "");
+            if (isLoadingStructureOnly) {
+                questionPanel.setQuestionHasResponses(false);
+            }
```

Comment:
In the case that it's loading only the structure, is it still building up the response rows, just not showing them?
Should `responses` be an empty list in that case, or `isShowingResponseRows` (in the called method) be set to `false`, or is this handled elsewhere already?